pkg/virt-config/virt-config.go

```diff
@@ -69,6 +72,9 @@ func getDefaultMachinesForArch() (string, string) {
 	if runtime.GOARCH == "ppc64le" {
 		return DefaultPPC64LEMachineType, DefaultPPC64LEEmulatedMachines
 	}
+	if runtime.GOARCH == "arm64" {
```

Comment:
Same here regarding to a parameter. Using a struct and having an arch member can make sense too. Up to you and the circumstances.